src/components/pages/Home/Notifications.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Badge, Dropdown, List, Menu } from 'antd';
+import { BellOutlined } from '@ant-design/icons';
+
+const NotificationListItem = props => {
+  const { notification, onClickView, onClickDismiss } = props;
+
+  const handleClick = key => {
+    return evt => {
+      evt.preventDefault();
+
+      if (key === 'view') {
+        return onClickView(notification.id);
+      } else if (key === 'dismiss') {
+        return onClickDismiss(notification.id);
+      }
+    };
+  };
+
+  return (
+    <List.Item
+      style={{
+        padding: '0.5rem',
+        marginBottom: '1rem',
+      }}
+      actions={[
+        <a
+          href="#"
+          style={{ marginRight: '0.5rem' }}
+          onClick={handleClick('view')}
+        >
+          View
+        </a>,
+        <a href="#" onClick={handleClick('dismiss')}>
```

Comment:
Since you are calling a method and passing a prop:
`onClick={evt => handleClick('dismiss')}`